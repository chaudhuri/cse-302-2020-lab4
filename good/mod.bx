// modulus

proc main() {
  var x = 42, y = -84 : int;
  print(y % x);
  print(x % y);
  var z = y % x : int;
  print(z);
  x = -201;
  y = 200;
  print(x % y);
  print(y % x);
}