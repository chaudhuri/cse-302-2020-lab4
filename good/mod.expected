0
42
0
-1
200
